specification/personal-demographics.yaml

```diff
@@ -569,18 +574,17 @@ paths:
           description: |
             An error occurred as follows:
             
-            | HTTP status | Error code                 | Description |
-            | ----------- | -------------------------- | --------------------------------------------- |
-            | 200         | TOO_MANY_MATCHES           | Too many matches were found - user should be told to refine their search parameters.	|
```

Comment:
is this definitely a 200? Not a typo where 200 should be 400 and it's in the correct place?

Reply:
No, it's a 200 - this is verified by the test in searchParams.feature called "Too many matches" - veit07 returns a 200 with the TOO_MANY_MATCHES message.